packages/gravity-ui-nunjucks/src/_patterns/03-organisms/00-page-structure/02-hero.njk

```diff
@@ -1,13 +1,15 @@
-{%- from 'atoms-cta' import cta -%}
-{%- from 'atoms-pullout' import pullout -%}
+{%- from 'atoms-image' import image -%}
 
 <section class="grav-c-hero">
-  <canvas></canvas>
-  {% if hero.pullout -%}
-    {{ pullout(hero.pullout) | safe }}
-  {%- endif %}
-  {% if hero.paragraph -%}
-    <p>{{hero.paragraph}}</p>
-  {%- endif %}
-  {{ cta(hero.cta.text, hero.cta.url) | safe }}
+  <div class="grav-c-hero__banner"></div>
```

Comment:
I thought about doing an image but the background stretching would become a problem unless there's something I don't know about CSS image stretching.

Reply:
I've been using `object-fit: cover` on the article cards `<img>` elements. It's not supported on IE10/11 so it would fall back to stretch to fit on those browsers 🤷‍♂

https://caniuse.com/#feat=object-fit

Reply:
Move to an `<img>` tag. Feels syntactically a lot more correct. Thanks for the tip